### PR TITLE
Changelog 1.19.2

### PR DESCRIPTION
**Pull request by jolheiser**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,45 @@ This changelog goes through all the changes that have been made in each release
...
+  * Vertical widths of containers removed (#24184) (#24211)
+  * Use correct locale key for forks page (#24172) (#24175)
+  * Sort repo topic labels by name (#24123) (#24153)
+  * Highlight selected file in the PR file tree (#23947) (#24126)
+* BUGFIXES
+  * Remove org users who belong to no teams (#24247) (#24313)
+  * Fix bug when deleting wiki with no code write permission (#24274) (#24295)
+  * Handle canceled workflow as a warning instead of a fail (#24282) (#24292)
+  * Load reviewer for comments when dismissing a review (#24281) (#24288)
+  * Only delete secrets belonging to its owner (#24284) (#24286)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Can you bump this up to security?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,45 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix calReleaseNumCommitsBehind (#24148) (#24197)
+  * Fix Org edit page bugs: renaming detection, maxlength (#24161) (#24171)
+  * Update redis library to support redis v7 (#24114) (#24156)
+  * Use 1.18's aria role for dropdown menus (#24144) (#24155)
+  * Fix 2-dot direct compare to use the right base commit (#24133) (#24150)
+  * Fix incorrect server error content in RunnersList (#24118) (#24121)
+  * Fix mismatch between hook events and github event types (#24048) (#24091)
+* BUILD
+  * Support converting varchar to nvarchar for mssql database (#24105) (#24168)
+* MISC
```
Comment (techknowlogick) [inline CHANGELOG.md]: I think you can rm this entire section

**Review by techknowlogick — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by wxiaoguang** [comment]
Don't set meta theme-color by default (#24340) #24346  ?

**Comment by jolheiser** [comment]
> Don't set meta theme-color by default (#24340) #24346 ?

Strange that doesn't show up in the generation. 🤔

**Comment by lunny** [comment]
#24369 merged

**Review by pat-s — COMMENTED** [review]
This Gitea version is the first to add "proper" support for the latest version of `redis-cluster` (due to the update of the redis go library and #23869 (which was in 1.19.1). This is a big step for high availability. Maybe worth mentioning it as a "feature" or at least "enhancement"?

**Comment by jolheiser** [comment]
> This Gitea version is the first to add "proper" support for the latest version of `redis-cluster` (due to the update of the redis go library and #23869 (which was in 1.19.1). This is a big step for high availability. Maybe worth mentioning it as a "feature" or at least "enhancement"?

I think this would be great to add to the blog post and gh release notes! I'll let you know when there's a blog pad and we can add it. 🥳

**Comment by wxiaoguang** [comment]
Do we need to figure out this ?

https://github.com/go-gitea/gitea/pull/24362#issuecomment-1525626200

**Review by delvh — APPROVED** [review]

**Comment by jolheiser** [comment]
Blocked by https://github.com/go-gitea/gitea/pull/24382

**Comment by yardenshoham** [comment]
Merged

**Pull request merged**